packages/orbit-components/src/Popover/components/ContentWrapper.tsx

```diff
@@ -297,60 +102,146 @@ const PopoverContentWrapper = ({
+    "--popper-position": popper.position,
+    "--zIndex": zIndex,
+    "--width": width,
+  } as React.CSSProperties;
 
   return (
     <>
-      <StyledOverlay shown={shown} onClick={onClose} />
-      {/* @ts-expect-error popper */}
-      <StyledPopoverParent
-        width={width}
+      <div
+        role="presentation"
+        className={cx(
+          "block",
+          "fixed",
+          "left-0 right-0 top-0",
+          "h-full w-full",
+          "bg-[rgba(0,0,0,0.5)]",
+          "duration-normal transition-[opacity,transform] ease-in-out",
+          "z-[999]",
+          "lm:hidden",
+          shown ? "opacity-100" : "opacity-0",
+        )}
+        onClick={onClose}
+      />
+      <div
+        role="tooltip"
         ref={popoverRef}
-        $zIndex={zIndex}
-        tabIndex={0}
         data-test={dataTest}
         id={id}
-        noPadding={noPadding}
-        overlapped={overlapped}
-        role="tooltip"
-        onKeyDown={handleKeyDown}
-        fixed={fixed}
-        shown={shown}
-        isInsideModal={isInsideModal}
-        {...popper}
+        className={cx(
+          "fixed",
+          "bottom-0 left-0 right-0",
+          "h-auto w-full",
+          "z-[1000]",
+          "box-border",
+          "shadow-raised-reverse",
+          "bg-elevation-raised-reverse",
+          "max-h-[calc(100%_-_theme(spacing.xl))]",
+          "focus:outline-none",
+          "lm:top-[var(--popper-top)]",
+          "lm:left-[var(--popper-left)]",
+          "lm:right-[var(--popper-right)]",
+          "lm:bottom-[var(--popper-bottom)]",
+          "lm:[position:var(--popper-position)]",
+          "lm:[transform:var(--popper-transform)]",
+          "lm:transition-[opacity] lm:duration-fast ease-in-out",
+          "lm:rounded-normal",
+          "lm:shadow-raised",
+          "lm:max-h-none",
+          isInsideModal ? "lm:z-[1000]" : "lm:z-[var(--zIndex)]",
+          width ? "lm:w-[var(--width)]" : "lm:w-auto",
+          shown ? "lm:opacity-100" : "lm:opacity-0",
+        )}
+        style={cssVars}
       >
-        <StyledPopoverContent ref={content} shownMobile={shown}>
-          <StyledContentWrapper
-            actionsHeight={actionsHeight}
+        <div
+          ref={content}
+          className={cx(
+            shown ? "translate-y-0" : "translate-y-full",
+            "will-change-transform",
+            "duration-fast transition-[opacity,transform] ease-in-out",
+            "lm:transform-none",
+            "lm-transition-none",
+          )}
+        >
+          <div
             ref={scrollingElementRef}
-            windowHeight={windowHeight}
-            maxHeight={maxHeight}
+            className={cx(
+              "overflow-auto",
+              "rounded-t-[theme(spacing.sm)]",
+              "absolute left-0",
+              "w-full",
+              "bg-white-normal",
+              "max-h-[calc(var(--window-height)-var(--actions-height)-32)]",
```

Comment:
We were checking if `windowHeight` and `actionsHeight` were defined. Shouldn't we need to check it here as well? We are defining fallback to 0, but that's not what we had before. Before, if one of these were not defined, it simply didn't have `max-height`. Now it will always have `max-height`, and even eventually possibly negative.